data/documentation.md

```diff
@@ -173,25 +172,25 @@ $E
 
 ##Answer summaries
 
-Use these to sum up content in the 'quick answer' format in 1 or 2 sentences at the top of the page by adding '$!' to the start and end of the text. Don't use this anywhere else, or in any other formats.
+We no longer add this markdown to any new content. (You may still come across it in the 'quick answer' format.) 
 
   $! This is an answer summary. $!
-
-There's an example of what this looks like on the [Child Benefit rates](https://www.gov.uk/child-benefit-rates) quick answer.
+  
+If you see this markdown, you can delete it.

```

Comment:
@raykhan66 Who told you this was the case? While the markdown just adds an extra gap at the moment, I was under the impression it was something we keep in case it's a class that gets styled in the future.